.browserslistrc

```diff
@@ -0,0 +1 @@
+IE 11
```

Comment:
we're starting to pick up a log of random config files on repo root. it's not a bad thing but some of them like this, jest, prettier can all be put to package.json

What are your thoughts on that ?

Reply:
I agree on having them on `package.json` to clean up the repo. But let's make a PR for doing that all in one